<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRename</class>
 <widget class="QDialog" name="DialogRename">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bulk rename</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/misc/polyphone.png</normaloff>:/misc/polyphone.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineText1">
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Overwrite existing name with key name as suffix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Overwrite existing name with key name and velocity as suffix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Overwrite existing name with numerical ascending suffix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Replace characters</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Insert at a specific position</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delete character range</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineText2">
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelString1">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Old</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelString2">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">New</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinPos2">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelPos">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinPos1">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>spinPos1</tabstop>
  <tabstop>spinPos2</tabstop>
  <tabstop>lineText1</tabstop>
  <tabstop>lineText2</tabstop>
  <tabstop>pushOk</tabstop>
  <tabstop>pushCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
